tests/integration/example/integration.sh

```diff
@@ -26,35 +26,40 @@ function process_result() {
     fi
 }
 
-# Build mixer binary
-bazel build //mixer/cmd/mixs:mixs
-MIXER_BINARY=$(pwd)/bazel-bin/mixer/cmd/mixs/mixs
+echo ${GOPATH}
 
-# Get fortio
-go get -u istio.io/fortio
+# Build mixer binary
+make mixs
+MIXER_BINARY=$(make where-is-out)/mixs
+ENVOY_BINARY=$(make where-is-out)/envoy
 
-# Download Proxy
+# Download Proxy Repo
 cd ..
 ls proxy || git clone https://github.com/istio/proxy
 cd proxy
-#git pull
-bazel build //src/envoy/mixer:envoy
-ENVOY_BINARY=$(pwd)/src/envoy/mixer/start_envoy
+git pull
+
+#ENVOY_BINARY=$(pwd)/usr/local/bin/envoy
+START_ENVOY=$(pwd)/src/envoy/mixer/start_envoy
 cd ../istio
 
+# Install Fortio
+cd vendor/istio.io/fortio
+make install
```

Comment:
I didn't notice this but it creates #3483 when using vendor - will fix

Reply:
Thanks